### ISSUE TITLE
Empty Stats: A11y

### ISSUE THREAD

**Issue by momo-ozawa** [wordpress-mobile/WordPress-iOS#17317]
## Description

All nudges should have appropriate a11y.

## Tasks
- [x] Add a11y to the new Grow Audience card
